source/main.c

```diff
@@ -380,14 +378,14 @@ int main()
     char *paths[2];
     int num_paths = 0;
 
-    for (int i = 0; i < NUM_SHORTCUTS; i++) {
+    for (int i = 1; i < NUM_SHORTCUTS; i++) {
       if (all_buttons_held & shortcuts[i].pad_buttons) {
         paths[num_paths++] = shortcuts[i].path;
         break;
       }
     }
 
-    paths[num_paths++] = default_path;
+    paths[num_paths++] = shortcuts[0].path;
```

Comment:
It would be nice if the special case could be removed entirely.
Move the default path to the bottom of the list, then adjust the condition in the loop.